src/map/status.c

```diff
@@ -12319,6 +12326,7 @@ void status_defaults(void) {
 	status->calc_mob_ = status_calc_mob_;
 	status->calc_pet_ = status_calc_pet_;
 	status->calc_pc_ = status_calc_pc_;
+	status->calc_pc_additional = status_calc_pc_additional;
```

Comment:
Instead of increasing (very very very little time by checking), why not the current method used??